tests/plot/test_scanpy_pl_api.py

```diff
@@ -0,0 +1,633 @@
+from pathlib import Path
+
+import matplotlib as mpl
+import pandas as pd
+import pytest
+
+mpl.use("Agg")
+
+import matplotlib.pyplot as plt
+
+plt.style.use("default")
+
+import os
+
+import numpy as np
+
+import ehrapy as ep
+
+CURRENT_DIR = Path(__file__).parent
+_TEST_IMAGE_PATH = f"{CURRENT_DIR}/_images"
+
+# Set NUMBA_CPU_NAME to "generic" before importing or using numba
```

Comment:
But why? The comment should explain WHY we're doing this because the code already tells us that we're doing just that.